src/components/entries/FEEL/FeelPopup.js

```diff
@@ -179,6 +180,14 @@ function FeelPopupComponent(props) {
     return () => emit('close', { domNode: popupRef.current });
   }, []);
 
+  useEffect(() => {
+
+    // Set focus on editor when popup is opened
+    if (editorRef.current) {
+      editorRef.current.focus();
+    }
+  }, [ editorRef ]);
```

Comment:
Specific focus is required now that the editor is not the first focusable element in the popup.

Reply:
:+1: